chain/chain/src/store.rs

```diff
@@ -761,6 +761,10 @@ impl ChainStore {
             }
         })
     }
+
+    pub fn get_rocksdb_statistics(&self) -> Option<String> {
```

Comment:
do we need this method on this level?

if yes -- we're 'leaking' the implementation details ('rocksdb') - which will make it harder in the future to use different storage system.

Reply:
+1, specifically, this leaks two things:

* name rocks (fixable by renaming to get_store_statisrics)
* the specific string format (fixable by parsing at the rocks layer and returning a `struct StoreStats(Map<String, String>)` )

Reply:
Done